doc/conf.py

```diff
@@ -120,13 +120,35 @@ def __getattr__(cls, name):
     "sphinx.ext.napoleon",
     "sphinx.ext.todo",
     "sphinx.ext.viewcode",
+    "sphinxext.opengraph",
     "sphinx_automodapi.automodapi",
     "sphinx_automodapi.smart_resolver",
     "m2r2",
 ]
 
 intersphinx_mapping = {"https://docs.pennylane.ai/en/stable/": None}
 
+# OpenGraph Metadata
+ogp_title = "Catalyst Documentation"
+ogp_type = "website"
+ogp_description = (
+    "Catalyst is an experimental package that enables just-in-time (JIT) "
+    "compilation of PennyLane programs. Compile the entire "
+    "quantum-classical workflow."
+)
```

Comment:
**Suggestion:** We should remove the `title` and `description` values since they don't appear as [options in the Open Graph extension](https://github.com/wpilibsuite/sphinxext-opengraph#options) and, as you pointed out below, seem to have no effect. (Also, `ogp_type` is `website` by default.)
```suggestion
```